libwebauthn/src/webauthn.rs

```diff
@@ -310,18 +315,21 @@ where
 {
     let get_info_response = channel.ctap2_get_info().await?;
     ctap2_request.handle_legacy_preview(&get_info_response);
-    let uv_proto = select_uv_proto(&get_info_response).await?;
-    let token_identifier = Ctap2AuthTokenPermission::new(
-        uv_proto.version(),
-        ctap2_request.permissions(),
-        ctap2_request.permissions_rpid(),
-    );
-    if let Some(uv_auth_token) = channel.get_uv_auth_token(&token_identifier) {
-        ctap2_request.calculate_and_set_uv_auth(&uv_proto, uv_auth_token);
-        Ok(UsedPinUvAuthToken::FromStorage)
-    } else {
-        user_verification_helper(channel, user_verification, ctap2_request, timeout).await
+    let maybe_uv_proto = select_uv_proto(&get_info_response).await;
```

Comment:
Instead of guarding here against `None`, we could probably just add the ability for `Ctap2AuthTokenPermission` to store an `Option<Ctap2PinUvAuthProtocol>` instead of a `Ctap2PinUvAuthProtocol`. This is basically only used as a lookup-key, so doesn't really depend on the UV-protocol being actually set.

Reply:
Yes, let's keep it the way it is!